packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -363,6 +364,12 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
     }
   }
 
+  // Return whether the contextual menu is hidden.
+  // Undefined value for hidden is equivalent to hidden being false.
```

Comment:
Great, thanks for this change! While you're here, could you also update the TSDoc comment for the `hidden` prop to specify default value as false?

Something like the following:

```
/**
   * If specified, renders the ContextualMenu in a hidden state.
   * ...
   * once the ContextualMenu is visible.
   * @defaultvalue false
   */
  hidden?: boolean;
```

Reply:
The default value for hidden is undefined.

Reply:
True - I guess the behavior acts as 'false', but it doesn't technically default to 'false'. Thanks for updating the comment!